types/qs/qs-tests.ts

```diff
@@ -10,7 +10,7 @@ qs.parse("a=b&c=d", { delimiter: "&" });
 (() => {
     let obj = qs.parse("a=z&b[c]=z&d=z&d=z&e[][f]=z");
     obj; // $ExpectType ParsedQs
-    obj.a; // $ExpectType string | ParsedQs | string[] | ParsedQs[] | undefined || string | string[] | ParsedQs | ParsedQs[] | undefined
+    obj.a; // $ExpectType string | ParsedQs | (string | ParsedQs)[] | undefined
```

Comment:
I simplified this to just the type reported by the test failure. I don't understand why there were two type expressions OR'd here, and the tests passed with this simplified expectation.